packages/hyperion-autologging-plugin-eventhash/src/index.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates. All Rights Reserved.
+ */
+
+'use strict';
+import type { ALChannelEvent } from "@hyperion/hyperion-autologging/src/AutoLogging";
+import { Channel } from "@hyperion/hyperion-channel";
+import * as XXH from "xxhashjs"
+
+export function init(channel: Channel<ALChannelEvent>): void {
+  const xxh = XXH.h64(0xABCD);
+  channel.addListener('al_ui_event', eventData => {
+    const data = `${eventData.event},${eventData.pageURI},${eventData.surface ?? ''},${eventData.reactComponentName ?? ''},${eventData.elementName ?? ''}` + null;
```

Comment:
One other thought I just had re-reviewing,  pageURI (full) may be quite unique because of query parameters or resource ids in URI path.  Should we consider cleaning those prior to hash?  I don't really want to add more processing on top of this but just thinking out loud.

Reply:
No, that's actually a good point. I think in the backend, we only look at the path, right?